bin/migrate-oats-data/noi/oats_to_alcs_notice_of_intent_table_etl/oats_to_alcs_notice_of_intent_table_etl.py

```diff
@@ -146,13 +148,18 @@ def map_basic_field(data):
 
 
 def map_prop_end_date(data):
-    date = data.get("nonfarm_use_end_date", "")
-    alcs_date = data.get("proposal_end_date", "")
-    if date is None and alcs_date is None:
-        return data
-    elif date is None:
-        return data
-    else:
+    data["proposal_end_date"] = None
+    data["proposal_end_date2"] = None
+    date = data.get("nonfarm_use_end_date", None)
+    noi_type = data.get("alr_change_code", None)
+
+    if date is not None:
```

Comment:
To confirm, End date 2 is null for everything except PFRS. Correct?

Reply:
correct, here is the reference to the db comment that I found
![image](https://github.com/bcgov/alcs/assets/61513701/1c1c2c2c-ebfb-4a45-901d-73a9ab3611a4)